sdk/js/src/index.ts

```diff
@@ -48,10 +58,12 @@ export interface WsOptions {
    * Max time to wait for a response from the server in milliseconds
    */
   response_timeout: number;
+  ping_interval: number;
 }
 
 const DEFAULT_WS_OPTIONS: WsOptions = {
   response_timeout: 10000,
+  ping_interval: 30000,
```

Comment:
I don't think you need to actively ping from client side. Server pings every subscriber, so you can make sure you are receiving a ping from server side every 30 seconds. You can look at swap-beacon for an example implementation

Reply:
I am embarrassed I didn't see this